AElf.Contracts.Consensus.DPoS.SideChain.Tests/AElf.Contracts.Consensus.DPoS.SideChain.Tests.csproj

```diff
@@ -8,8 +8,12 @@
         <PackageReference Include="coverlet.msbuild" Version="2.5.1" />
         <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" />
         <PackageReference Include="Shouldly" Version="3.0.2" />
+        <PackageReference Include="Volo.Abp.Core" Version="0.14.0" />
```

Comment:
<PackageReference Include="Volo.Abp.Core" Version="0.14.0" />
don't add the ref

Reply:
removed volo reference.